R/feature.align.R

```diff
@@ -114,7 +114,19 @@ create_rows <- function(features,
 }
 
 
-create_aligned_feature_table <- function(all_table,
+#' Align peaks from spectra into a feature table.
+#'
+#' @param features_table A list object. Each component is a matrix which is the output from compute_clusters().
+#' @param min_occurrence  A feature has to show up in at least this number of profiles to be included in the final result.
+#' @param sample_names list List of sample names.
+#' @param mz_tol_relative The m/z tolerance level for peak alignment. The default is NA, which allows the
+#'  program to search for the tolerance level based on the data. This value is expressed as the
+#'  percentage of the m/z value. This value, multiplied by the m/z value, becomes the cutoff level.
+#' @param rt_tol_relative The retention time tolerance level for peak alignment. The default is NA, which
+#'  allows the program to search for the tolerance level based on the data.
+#'
```

Comment:
Description of return values still has to be added.

Reply:
Done in [3b59a5b](https://github.com/RECETOX/recetox-aplcms/pull/154/commits/3b59a5ba4abef27e8319e262d1a9ed7293b48383).